react-project/src/components/Visualization.jsx

```diff
@@ -100,7 +99,6 @@ function App() {
         }
         batch(() => {
           dispatch(scopeMainTreemapIn(payloadGenerator("path", urlDataPath)));
-          dispatch(scopeMiniTreemapIn(payloadGenerator("path", urlDataPath)));
```

Comment:
And also this one? This line and line 90 ensure that the simulation mode treemap gets updated with the main treemap. Maybe there is a corresponding change in the reducer for this, I should check